<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowClientes</class>
 <widget class="QMainWindow" name="WindowClientes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Clientes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Endereco</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Profissao</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Salario</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_nome">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_endereco">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>351</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_profissao">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="input_salario">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_retroceder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_avancar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_salvar">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>479</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCadastros">
    <property name="title">
     <string>Cadastros</string>
    </property>
    <addaction name="actionNovo_Cliente"/>
    <addaction name="actionRemover_Client"/>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Sobre</string>
    </property>
    <addaction name="actionVersao"/>
   </widget>
   <addaction name="menuCadastros"/>
   <addaction name="menuSobre"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNovo_Cliente">
   <property name="text">
    <string>Novo Cliente</string>
   </property>
  </action>
  <action name="actionRemover_Client">
   <property name="text">
    <string>Remover Cliente</string>
   </property>
  </action>
  <action name="actionVersao">
   <property name="text">
    <string>Versao</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
